cypress/integration/chi-custom-elements/icon.spec.js

```diff
@@ -0,0 +1,58 @@
+/// <reference types="Cypress" />
+
+describe('Icons', function() {
+
+  beforeEach(()=>{
+    cy.fixture('chidata.json').as('chidata');
+  });
+
+  it('Base icon should be included inside the custom element', function() {
+    cy.visit('tests/custom-elements/icons.html');
+
+    cy.get('[data-cy="icon-single"]')
+      .should('have.class', 'hydrated')
+      .children()
+      .first()
+      .should('match', 'div.a-icon')
+      .children()
+      .first()
+      .should('match', 'svg');
+
+  });
+
+  it('Sized icons should have appropriate class', function() {
+
+    cy.get('[data-cy="icon-size"]')
+      .should('have.class', 'hydrated')
+      .children()
+      .first()
+      .should('match', 'div.a-icon.-xl')
+      .and('not.have.class', '-sm')
+      .children()
+      .first()
+      .should('match', 'svg');
+
+  });
+
+  it('Colored icons should have appropriate class', function() {
+
+    cy.get('[data-cy="icon-color"]')
+      .should('have.class', 'hydrated')
+      .children()
+      .first()
+      .should('match', 'div.a-icon.-text--primary')
```

Comment:
@dani-cl-madrid Can you check the test colors classes again because i found a bug in tests related custom-elements and standard tests icons colors, please? the color classes must be as per image and the test don't match some classes as -text--secondary and -text--white and instead exist -text--inverse 
and it is wrong. <img width="409" alt="Screen Shot 2019-05-21 at 4 05 44 PM" src="https://user-images.githubusercontent.com/43439742/58104354-f41fde80-7be4-11e9-8637-2eb09b3bc9c3.png">

Reply:
This is not related to this PR. @davidojedacl has opened a new ticket to add new text-colors to all icons documentation and test. Please @jllr , merge as soon as you receive ACK from @davidojedacl . TY.

Reply:
ACK